db/alembic/versions/831ad1e71a80_add_ipaddress_column_to_daily_usage_.py

```diff
@@ -0,0 +1,25 @@
+"""Add IPAddress column to daily_usage table
+
+Revision ID: 831ad1e71a80
+Revises: 32753a3e09e0
+Create Date: 2025-08-21 08:57:10.248625
+
+"""
+
+from typing import Sequence, Union
+
+# revision identifiers, used by Alembic.
+revision: str = "831ad1e71a80"
+down_revision: Union[str, None] = "32753a3e09e0"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    """Upgrade schema."""
```

Comment:
There's nothing in the upgrade?

Reply:
No clue how that happened! Just pushed an updated version of the file